### PR TITLE
muse: update to 4.1.0.

### PR DESCRIPTION
**Pull request by unspecd**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - ppc64le-musl

### CODE REVIEW

**Comment by paper42** [comment]
For the i686 build, please import muse-sequencer/muse#1067

**Comment by unspecd** [comment]
Done. Thanks for the suggestion.

File: srcpkgs/muse/template [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Template file for 'muse'
 pkgname=muse
-version=4.0.0
-revision=2
+version=4.1.0
+revision=1
 wrksrc="muse-${version}"
```
Comment (paper42) [inline srcpkgs/muse/template:5]: $wrksrc shouldn't be necessary

**Review by paper42 — COMMENTED** [review]
I would call the patch i686.patch to make its purpose obvious.

File: srcpkgs/muse/template (hunk 1 above)
Comment (unspecd) [reply srcpkgs/muse/template]: Fixed.

**Pull request merged**